packages/core/src/domain/telemetry/telemetryEvent.types.ts

```diff
@@ -383,6 +383,14 @@ export type TelemetryConfigurationEvent = CommonTelemetryProperties & {
         name: string
         [k: string]: unknown
       }[]
+      /**
+       * Whether the SDK is initialised on the application's main or a secondary process
+       */
+      is_main_process?: boolean
+      /**
+       * The list of events that include feature flags collection
+       */
+      collect_feature_flags_on?: ('view' | 'error' | 'vital')[]
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I don’t think customers should be able to opt-out from having feature flags on views as feature flag tracking (https://www.datadoghq.com/dg/real-user-monitoring/feature-flag-tracking/?utm_source=google&utm_medium=paid-search&utm_campaign=dg-rum-emea-feature-flags&utm_keyword=feature%20flags%20tracking&utm_matchtype=b&igaag=153979902437&igaat=&igacm=14746103004&igacr=675182438154&igakw=feature%20flags%20tracking&igamt=b&igant=g&utm_campaignid=14746103004&utm_adgroupid=153979902437&gad_source=1&gclid=Cj0KCQiA0--6BhCBARIsADYqyL-7A4bmovzzHp49SZ40aNHBCp-HofNZDn355E8wGk7HcDGoEgfeMhwaAnebEALw_wcB) rely on it.